### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Fixed Typo, Grammar, label names, doc structure, and information flow issues
- Rewrote the usage section linking to the FortiSOAR documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,24 @@
 - **Version**:  1.0.0
 - **Certified**: Yes
 - **Publisher**: Fortinet
-- **Compatible Version**: FortiSOAR cloud instance
+- **Compatible Version**: FortiSOAR&trade; cloud instance
```
Comment (cs-suyog-jinturkar) [inline README.md]: Fortisoar cloud instance or FortiSOAR cloud? Please check with other docs

File: docs/contents.md [hunk 2]
```diff
@@ -42,10 +43,15 @@ When creating notification rules, selecting **Manual Input** as a notification t
 ## Playbook Collection
 
 | 02 - Use Case - FortiSOAR for Microsoft Teams |
-|:------------------------------------|
+|:----------------------------------------------|
+
+| Playbook Name                  | Description                                                                                                                    |
+|:-------------------------------|:-------------------------------------------------------------------------------------------------------------------------------|
+| Create Indicator               | Extract Indicators from the text provided as Input                                                                             |
+| Create Alert                   | Creates an alert based on the values provided in the form                                                                      |
```
Comment (cs-suyog-jinturkar) [inline docs/contents.md]: All three PBS are triggered from Ms teams as part of FSR command exec. We can have consistent description for them...

File: docs/contents.md [hunk 3]
```diff
@@ -42,10 +43,15 @@ When creating notification rules, selecting **Manual Input** as a notification t
...
+| Enrich IP From Microsoft Teams | Reference playbook that gets triggered from Microsoft Teams based and provides latest reputation formatted for Microsoft Teams |
```
Comment (cs-suyog-jinturkar) [inline docs/contents.md]: This description should be changed to "Reference playbook that gets triggered and provides latest well formatted reputation using configured threat Intel platform"

File: docs/setup.md [hunk 4]
```diff
@@ -1,27 +1,50 @@
...
+
+    5. [Configuring the FortiSOAR For Microsoft Teams Application in your FortiSOAR instance](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#configureFSRForMsTeamsAppinFSR)
+
+    After completing the set up of FortiSOAR&trade; for Microsoft Teams app on Azure, configure the Microsoft Teams connector.
+
+    - To configure and use Microsoft Teams connector, refer to [Configuring Microsoft Teams Connector](https://docs.fortinet.com/fortisoar/connectors/ms-teams)
+
+    The bridge between FortiSOAR&trade; with Microsoft Teams is now ready for end-to-end communication between FortiSOAR&trade; and Microsoft Teams.
+
+- For optimal performance of **FortiSoar for Microsoft Teams** solution pack, you can install and configure the following connectors:
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: FortiSOAR

File: docs/usage.md [hunk 5]
```diff
@@ -1,43 +1,81 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Usage
 
-This section gives a brief description on some of the commands used in FortiSOAR for Microsoft Teams application:
+After completing the set up of FortiSOAR&trade; for Microsoft Teams application, you can begin using the app as described in the [FortiSOAR&trade;-Microsoft Teams Application Usage](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#FSRMsTeamsUsage) section of FortiSOAR&trade; product documentation.
+
+This section gives a brief description on some of the commands used in FortiSOAR&trade; for Microsoft Teams application.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Some?? we mentioned all supported cmds

File: docs/usage.md [hunk 6]
```diff
@@ -1,43 +1,81 @@
...
 
 ## `@fortiSOAR createAlert`
 
 This commands performs the following actions:
 
-1. Displays a new alert form.
-2. Uses the details from the form to create a new alert in FortiSOAR.
+1. Displays a **Create New Alert** form.
+2. Takes the name and description entered in the manual input form on the MS Teams app to create a new alert in FortiSOAR&trade;.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Please use consistent language like "Microsoft Teams application"

File: docs/usage.md [hunk 7]
```diff
@@ -1,43 +1,81 @@
...
+
+This section gives a brief description on some of the commands used in FortiSOAR&trade; for Microsoft Teams application.
 
 ## `@fortiSOAR createAlert`
 
 This commands performs the following actions:
 
-1. Displays a new alert form.
-2. Uses the details from the form to create a new alert in FortiSOAR.
+1. Displays a **Create New Alert** form.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:14]: Form or adaptive card or manual input form or manual input form aka adaptive card?

Also, we need to use it consistently everywhere

File: docs/usage.md [hunk 8]
```diff
@@ -1,43 +1,81 @@
...
+
 ## `@fortiSOAR createIndicator`
 
 This commands performs the following actions:
 
-1. Displays a new indicator form.
-2. Uses the details from the form to create a new indicator in FortiSOAR.
-3. Takes an indicator value in the command to create the indicator with the specified value.
+1. Displays a **Create New Indicator** form.
+2. Takes the indicator value from the command to create an indicator in FortiSOAR&trade; with the information specified in the input form on the MS Teams app.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Same as above

File: docs/usage.md [hunk 9]
```diff
@@ -1,43 +1,81 @@
...
 
-    `@fortiSOAR createIndicator <indicator_value>` adds an indicator in FortiSOAR and gets the latest enrichment details back to Microsoft Teams within seconds.
+### Example
 
-    For example, `@fortiSOAR createIndicator gumblar.cn` creates an indicator of value `gumblar.cn`.
+```
+@fortiSOAR createIndicator gumblar.cn
+```
+
+This command adds an indicator of value `gumblar.cn` in FortiSOAR&trade;
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Add or create?

Comment (cs-suyog-jinturkar) [reply docs/usage.md]: Please use consistently

File: docs/usage.md [hunk 10]
```diff
@@ -1,43 +1,81 @@
...
 
 You can create indicators on the fly for enrichment and further investigation by the Security Operations Center (SOC) team.
 
+For more information and examples for this command, refer to the [Adding an indicator with the indicator value specified](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#Adding_an_indicator_with_the_indicator_value_specified) section of FortiSOAR&trade; product documentation.
+
 ## `@fortiSOAR invokePlaybook`
 
-This command takes another argument `<tag_name>` to trigger the playbook containing the tag `<<tag_name>>`.
+This command takes another argument `<tag_name>` to trigger the playbook containing the tag `<tag_name>`.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: FortiSOAR pb or pb in FortiSOAR

File: docs/usage.md [hunk 11]
```diff
@@ -1,43 +1,81 @@
...
+
+> **NOTE**:  If there are multiple playbooks with the same tag, the first found playbook is executed.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Executed or triggered?? Consistency is missing

File: docs/usage.md [hunk 12]
```diff
@@ -1,43 +1,81 @@
...
+
+To create playbooks that can be triggered from Microsoft Teams chat interface, refer to [Create a custom playbook to be triggered from Microsoft Teams](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0How_to_create_a_custom_playbook_that_can_be_triggered_from_Microsoft_Teams_) section of FortiSOAR&trade; product documentation.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:52]: Chat or App?

File: docs/usage.md [hunk 13]
```diff
@@ -1,43 +1,81 @@
...
+@fortiSOAR invokePlaybook <tag_name>
+```
+
+### Example
+
+```
+@fortiSOAR invokePlaybook getIPRep
+```
+
+The command invokes the playbooks with the tag `getIPRep`.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Invokes or triggers??

File: docs/usage.md [hunk 14]
```diff
@@ -1,43 +1,81 @@
...
+
+>**NOTE**: The playbook being invoked must have a `bot_enabled` tag.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: The playbook being invoked must have a `bot_enabled` tag.  --> The playbook being invoked must ALSO have a `bot_enabled` tag.

File: docs/contents.md [hunk 15]
```diff
@@ -3,49 +3,55 @@
...
+| Name            | Description                                                                                                                                                                                                                                              |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Microsoft Teams | Microsoft Teams is a chat-based workspace in Office 365 that provides global, remote, and dispersed teams with the ability to work together and share information using a common space. This connector facilitates automated operation related to teams. |
+| VirusTotal      | Scans and analyzes suspicious files, URLs, IP addresses and retrieves reports from VirusTotal about them                                                                                                                                                 |
```
Comment (cs-abhinav-goel) [inline docs/contents.md]: With this SP, no Virus Total & IPStack connector gets Installed.

File: docs/contents.md [hunk 16]
```diff
@@ -3,49 +3,55 @@
...
 
 ## Notification Channel
 
-| Name               | Description                                         |
-|:-------------------|:----------------------------------------------------|
-| Microsoft Teams Link | To send a message to Microsoft Teams when certain rule is met. |
-| Microsoft Team      | It Sends a Manual Input Form to the Microsoft Teams application.|
+| Name                 | Description                                                   |
+|:---------------------|:--------------------------------------------------------------|
+| Microsoft Teams Link | Sends a message to Microsoft Teams when certain rule is met.  |
```
Comment (cs-abhinav-goel) [inline docs/contents.md]: @cs-abhishek-shukla : Can we have some other wording. 
Like "Sends a message to Microsoft Teams having link to the Manual Input when a rule using this channel is triggered"

File: docs/contents.md [hunk 17]
```diff
@@ -3,49 +3,55 @@
...
+| Microsoft Teams      | Sends a inline interactive form to the Microsoft Teams application. |
```
Comment (cs-abhinav-goel) [inline docs/contents.md]: @cs-abhishek-shukla : Similarly for this one. @shivaji can help you with this one.

File: docs/contents.md [hunk 18]
```diff
@@ -3,49 +3,55 @@
...
+| Microsoft Teams > Notify on External Manual Input Form | Sends an inline interactive form to the Microsoft Teams application when the manual input step for Microsoft Teams is triggered. |
+| Microsoft Teams > Notify on Playbook failure           | Sends an error when a playbook with a `bot_enabled` tag fails.                                                                   |
+| Microsoft Teams > Send Manual Input link               | Sends a link to Microsoft Teams to click and open the manual input form on FortiSOAR&trade;.                                     |
 
 When creating notification rules, selecting **Manual Input** as a notification trigger, makes following conditions available:
-- **External Channels (Inline)** - The manual input form appears inline with other FortiSOAR messages in Microsoft Teams.
+
+- **External Channels (Inline)** - The inline interactive form appears with other FortiSOAR&trade; messages in Microsoft Teams.
 
     ![](./res/inline-form.png)
```
Comment (cs-abhinav-goel) [inline docs/contents.md:37]: Need to update the Image, as in the current Every line refer below image
image

File: docs/contents.md [hunk 19]
```diff
@@ -3,49 +3,55 @@
...
 
 | 02 - Use Case - FortiSOAR for Microsoft Teams |
-|:------------------------------------|
+|:----------------------------------------------|
+
+| Playbook Name                  | Description                                                                                                                                                |
+|:-------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Create Indicator               | Extracts indicators from the text provided as input                                                                                                        |
+| Create Alert                   | Creates an alert based on the values provided in the form                                                                                                  |
+| Enrich IP From Microsoft Teams | A reference playbook triggered from Microsoft Teams chat interface and provides latest reputation information with formatting in-line with Teams interface |
```
Comment (cs-abhinav-goel) [inline docs/contents.md]: @cs-abhishek-shukla : The Playbooks for "Enrich IP From Microsoft Teams" is incorrect.

We have below Playbook 
>"Enrich IP"  | Gets triggered on enrichIP command when used in Microsoft Teams
>"Enrich IP > Enrichment" | Reference playbook that gets triggered from Microsoft Teams based and provides latest reputation formatted for Microsoft Teams

@cs-abhishek-shukla

**Review by cs-abhinav-goel — CHANGES_REQUESTED** [review]

**Review by cs-abhinav-goel — APPROVED** [review]
Looks good from my side

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**